portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,41 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
+ * @returns none
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+    ["I have two younger brothers both about a foot taller than me.", 
+     "My parents are from Vietnam but I don't speak any Vietnamese.",
+     "I have never been to anywhere else in Australia except Melbourne, where I was born and raised."
+    ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
 }
+
+/**
+ * Show the full blog post text for blog post div with the given ID.
+ * @param {*} blogId ID of the div of the blog post to be shown.
```

Comment:
nit: We should include the param type within the `{}`

Reply:
Done! I was wondering what the { } was for! 😅

Reply:
Thanks! You can look it, and other style guide suggestions, here: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments